styles/header.css

```diff
@@ -53,18 +82,62 @@
   }
 }
 
+/* HRs in the navbar have to be positioned vertically relative to the nav lists,
+ * but horizontally relative to the entire container. They aren't valid as
+ * children of a <ul>, so they have to be siblings, and they can't be inside
+ * an <li> because they would be counted as items in the list. Repositioning of
+ * other elements during opening of the menu makes them jump around, so we also
+ * have to disable animation of opening/closing the nav. */
+.navbar hr {
```

Comment:
Where does this occur?

Reply:
https://github.com/sul-dlss/component-library/blob/a13b573268bfaadb8082020c5430aec1a34e23d5/header/with_navigation_links.html#L223 and https://github.com/sul-dlss/component-library/blob/a13b573268bfaadb8082020c5430aec1a34e23d5/header/with_navigation_links.html#L308